presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -443,4 +458,33 @@ private static double percentage(double count, double total)
         }
         return min(100, (count * 100.0) / total);
     }
+
+    private class PreviousStageStatus
```

Comment:
This class is used for both Stage Status and Statement Status. How about just call it `Status` since it's a internal  private class? Another thought is `ProgressStatus`. @martint @raghavsethi